examples/nextjs-cldimage/package-lock.json

```diff
@@ -0,0 +1,4170 @@
+{
```

Comment:
ideally we wouldn't check this file in. im using pnpm to manage the overall repository as a monorepo, so if someone wants to use npm to run a project, they _can_ but from a maintanence perspective, pnpm will handle everything

Reply:
Oh okay. I thought of using pnpm too but because this example looks like it's available for everyone, i went ahead with `npm i`

Will update and push the pnpm lockfile